CHANGES.rst

```diff
@@ -15,7 +15,7 @@ Imviz
 ^^^^^
 - Fix a bug where footprints did not overlay when created via API. [#2790]
 
-- Improved behavior when orientations are created or selected without having data loaded in the viewer. [#2789]
+- Improved behavior when orientations are created or selected without having data loaded in the viewer. [#2808]
```

Comment:
I think the bot just leaves this referencing the original PR and then just add the no changelog entry needed label

Reply:
Makes sense, fixed!